pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -1041,24 +1063,24 @@ created with @racket[thread/suspend-to-kill] can have zero
 custodians. Extra custodians become associated with a thread through
 @racket[thread-resume] (see @secref["threadkill"]). When a thread
 has multiple custodians, it is not necessarily killed by a
-@racket[custodian-shutdown-all], but shut-down custodians are removed
-from the thread's managing set, and the thread is killed when its
+@racket[custodian-shutdown-all]. Instead, shut-down custodians are removed
```

Comment:
+1 for keeping ", but" or changing to ", but rather"

Reply:
Sorry, I don't understand what you are indicating here.